### PR TITLE
support capture zk packet from loopback device and ignore null packet

### PR DESCRIPTION
**Pull request by sel-fish**
1. support capture zk packet from loopback device
2. ignore packet which has no user data

### CODE REVIEW

File: zktraffic/base/sniffer.py [hunk 1]
```diff
@@ -221,13 +221,14 @@ def handle_packet(self, packet):
       message = self.message_from_packet(packet)
       self.handle_message(message)
     except (BadPacket, StringTooLong, DeserializationError, struct.error) as ex:
+      print("got: %s" % str(ex))
```
Comment (Yasumoto) [inline zktraffic/base/sniffer.py]: I think it's only intended to be verbose if the config is set to dump out "bad packets" so mind deleting this?

Comment (sel-fish) [inline zktraffic/base/sniffer.py]: @Yasumoto sorry for that, forgot to remove the debug message. fix it already.

**Comment by Yasumoto** [comment]
Thanks @sel-fish !

**Comment by sel-fish** [comment]
@Yasumoto the honor is mine :)

**Pull request merged**